src/server/engine_shard_set.cc

```diff
@@ -836,7 +837,7 @@ void EngineShardSet::Init(uint32_t sz, bool update_db_time) {
                  << HumanReadableNumBytes(256_MB * shard_queue_.size()) << ". Exiting..";
       exit(1);
     }
-    is_tiering_enabled_ = true;
```

Comment:
why did you change this?

Reply:
to be able to check is tiering is enabled without holding any object of engine shard set

Reply:
shard_set is globally available object. why it matters?

Reply:
you are right. I reverted this change